spec/services/post_scraper_spec.rb

```diff
@@ -10,13 +10,13 @@
     url = 'http://wild-pegasus-appeared.dreamwidth.org/403.html?view=flat'
     scraper = PostScraper.new(url)
     expect(scraper.url.sub('view=flat', '')).not_to include('view=flat')
```

Comment:
nit (not your fault): this is more readable as `expect(scraper.url.scan(/view=flat/).length).to eq(1)`